### PR TITLE
Feaute/2000

### PR DESCRIPTION
**Pull request by uedamiku**
管理者画面でトッピング一覧が表示されるようにしました。
削除と追加はできない。

### CODE REVIEW

File: src/main/java/com/example/controller/AdminController.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 import com.example.domain.Item;
 import com.example.domain.LoginUser;
+
+import com.example.domain.Topping;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.example.domain.Topping'

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.Topping' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/AdminController.java [hunk 2]
```diff
@@ -49,8 +50,9 @@ public String showTopPage(Model model, @AuthenticationPrincipal LoginUser loginU
 	@GetMapping("/show-item-list")
 	public String showItemList(Model model) {
 		List<Item> showItemList = itemListService.showItemList("admin");
-		System.out.println(showItemList);
+		List<Topping> toppingItemList = adminService.showToppingList();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 3]
```diff
@@ -49,8 +50,9 @@ public String showTopPage(Model model, @AuthenticationPrincipal LoginUser loginU
...
 		model.addAttribute("itemList", showItemList);
+		model.addAttribute("toppingItemList", toppingItemList);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:55]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:55]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/service/AdminService.java [hunk 4]
```diff
@@ -11,9 +11,13 @@
 import org.springframework.web.multipart.MultipartFile;
 
 import com.example.domain.Item;
+
+import com.example.domain.Topping;
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.example.domain.Topping'

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.Topping' import. Should be before 'org.springframework.web.multipart.MultipartFile'.

File: src/main/java/com/example/service/AdminService.java [hunk 5]
```diff
@@ -11,9 +11,13 @@
...
 import com.example.domain.User;
 import com.example.repository.ItemRepository;
 import com.example.repository.OrderRepository;
+import com.example.repository.ToppingRepository;
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.repository.ToppingRepository' import. Should be before 'org.springframework.web.multipart.MultipartFile'.

File: src/main/java/com/example/service/AdminService.java [hunk 6]
```diff
@@ -32,6 +36,9 @@ public class AdminService {
 	@Autowired
 	UserRepository userRepository;
 
+	@Autowired
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:39]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:39]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/service/AdminService.java [hunk 7]
```diff
@@ -32,6 +36,9 @@ public class AdminService {
...
+	ToppingRepository toppingRepository;
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 8]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
 		return userList;
 	}
 
+	/**
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 9]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+	 * 全トッピング情報を取得します.
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 10]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+	 *
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 11]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
 		return userList;
 	}
 
+	/**
+	 * 全トッピング情報を取得します.
+	 * 
+	 * @return トッピング情報
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@return' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 12]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+	 */
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/service/AdminService.java [hunk 13]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+	public List<Topping> showToppingList() {
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/service/AdminService.java [hunk 14]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+		List<Topping> toppingList = toppingRepository.findAll();
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/service/AdminService.java [hunk 15]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+		return toppingList;
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/service/AdminService.java [hunk 16]
```diff
@@ -52,6 +59,16 @@ public List<User> showUserList() {
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/service/AdminService.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**